packages/-ember-data/tests/integration/record-array-manager-test.js

```diff
@@ -256,11 +256,11 @@ module('integration/record_array_manager', function(hooks) {
     assert.deepEqual(internalModel._recordArrays.toArray(), [recordArray]);
   });
 
-  test('liveRecordArrayFor always return the same array for a given type', function(assert) {
+  test('liveRecordArrayFor always return the same array for a given type', async function(assert) {
     assert.equal(manager.liveRecordArrayFor('foo'), manager.liveRecordArrayFor('foo'));
   });
 
-  test('liveRecordArrayFor create with content', function(assert) {
+  test('liveRecordArrayFor create with content', async function(assert) {
```

Comment:
Why are some of these changed to `async` functions without any `await`'s in them?

Reply:
it prevents leakage for unsettled things we don't care about, which is fairly common. It effectively adds `await settled()` to every test.

Reply:
After OOB discussion @rwjblue has opened up this PR to enable us to not add unnecessary `async` tests: https://github.com/emberjs/ember-qunit/pull/528